graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx

```diff
@@ -125,4 +139,7 @@ class EventNotificationForm extends React.Component {
   }
 }
 
-export default EventNotificationForm;
+export default connect(EventNotificationForm, {
```

Comment:
Not sure about this, keeping the state in a store might not be the best option. Maybe @dennisoelkers  knows some better solution?

Reply:
He did indeed :-)